lfroms/COMP4109-Project
# Set up basic NodeJS server

### What is the problem being solved in this PR?
This PR adds a basic NodeJS server that returns `Hello World`, as well as a GitHub Action for linting the source code on every commit.

### Related issues or PRs
Resolves #3

### How did you accomplish this and why did you choose this approach?
- Standard method for setting up a NodeJS server.

### How to test changes
1. Clone this repo.
2. `yarn install`
3. `yarn start:dev`
4. Go to `localhost:3000` and ensure that you see the `Hello World!` message.

### Checklist
